apps/src/applab/commands.js

```diff
@@ -1544,13 +1537,29 @@ applabCommands.handleReadValue = function(opts, value) {
   }
 };
 
+applabCommands.getKeyValueSync = function(opts) {
+  apiValidateType(opts, 'getKeyValueSync', 'key', opts.key, 'string');
+  var onSuccess = applabCommands.handleGetKeyValueSync.bind(this, opts);
+  var onError = applabCommands.handleGetKeyValueSyncError.bind(this, opts);
+  AppStorage.getKeyValue(opts.key, onSuccess, onError);
+};
+
+applabCommands.handleGetKeyValueSync = function(opts, value) {
+  opts.callback(value);
+};
+
+applabCommands.handleGetKeyValueSyncError = function(opts, message) {
+  // Call callback with no value parameter (sync func will return undefined)
+  opts.callback();
+  Applab.log(message);
+};
+
```

Comment:
Are the handlers available as top level commands? If not, should we just have them be local functions? It feels like applabCommands should consist only of things that I expect to call from the editor as a user.